src/test/java/com/github/kiulian/downloader/YoutubeSearchExtractor_Tests.java

```diff
@@ -0,0 +1,105 @@
+package com.github.kiulian.downloader;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.*;
+
+import com.github.kiulian.downloader.downloader.request.RequestSearchContinuation;
+import com.github.kiulian.downloader.downloader.request.RequestSearchResult;
+import com.github.kiulian.downloader.downloader.response.Response;
+import com.github.kiulian.downloader.model.search.*;
+import com.github.kiulian.downloader.model.search.field.SortField;
+import com.github.kiulian.downloader.model.search.field.TypeField;
+
+@DisplayName("Tests extracting metadata from search results")
+public class YoutubeSearchExtractor_Tests {
+
+    private YoutubeDownloader downloader;
+
+    @BeforeEach
+    void initDownloader() {
+        downloader = new YoutubeDownloader();
+    }
+
+    @Test
+    @DisplayName("search 'nasa' should have many results")
+    void searchNasaAndContinuation_Success() {
+        assertDoesNotThrow(() -> {
+            SearchResult result = search(new RequestSearchResult("nasa"));
+            assertTrue(result.estimatedResults() > 20_000_000, "Estimated results should be over 20 M");
+            assertTrue(!result.channels().isEmpty(), "Result should contain a channel");
```

Comment:
this and further calls `assertTrue(!...)`can be replaced with `assertFalse(...)` ?

Reply:
Yes, unbelievable.
I tried to import the method but was not able to do so for a reason, and forgot to change it.